apps/mobile/src/main.tsx

```diff
@@ -1,4 +1,5 @@
 import { AppRegistry } from 'react-native';
 import AppRoot from './app/App';
+import { name } from '../app.json';
 
-AppRegistry.registerComponent('inHEARTive', () => AppRoot);
+AppRegistry.registerComponent(name, () => AppRoot);
```

Comment:
Check @pmusialik-intive PR, it's related I believe. app.json file should be updated to inHEARTive if we solve it this way. Android app was renamed to inHEARTive and the change in registerComponent() comes from that. Make sure it won't break Android build (not only emulator run but actual build used for example in Github Actions).

Reply:
Here We just get name from json. I just improve this.